vercel.json

```diff
@@ -12,8 +12,8 @@
       }
     },
     {
-      "src": "/favicon.ico",
-      "dest": "/favicon.ico"
+      "src": "/(.*).ico",
+      "dest": "/$1.ico"
```

Comment:
The route here is ambiguity, it will break the url looks like:

https://github1s-git-fork-shaedil-master-vscode-github1s.vercel.app/conwnet/github1s/blob/master/resources/favicon.ico

May be we can replace it with `/favicon-(dark|light).ico)`

And is it necessary for us to leave a default favicon.ico?

Reply:
+1, sorry I didn't think of the `https://github1s-git-fork-shaedil-master-vscode-github1s.vercel.app/conwnet/github1s/blob/master/resources/favicon.ico` URL. 

I think we might want to keep the default `favicon.ico` since browsers tend to request the file anyway.

Reply:
Browsers will always request `favicon.ico` unless there are icons present via `<link>`, in that case, the favicon switcher will set the correct favicon.

Reply:
Sounds good, thanks for the explanation. Could you please update the routes based on @conwnet 's suggestion?